regexes/device/mobiles.yml

```diff
@@ -22964,6 +23014,22 @@ TurboKids:
   device: 'tablet'
   model: '$1'
 
+# Turkcell (https://www.turkcell.com.tr/)
+Turkcell:
+  regex: 'TURKCELL'
```

Comment:
```yml
regex : 'TURKCELL[_ ]'
```
* Mozilla/5.0 (iPhone; CPU iPhone OS 7_1_2 like Mac OS X) AppleWebKit/537.51.2 (KHTML, like Gecko) Mobile/11D257 [FBAN/MessengerForiOS;FBAV/84.0.0.25.71;FBBV/36443639;FBRV/0;FBDV/iPhone3,2;FBMD/iPhone;FBSN/iPhone OS;FBSV/7.1.2;FBSS/2;FBCR/Turkcell;FBID/phone;FB
* Mozilla/5.0 (Android 6.0.1; SM-J700F Build/MMB29K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3317.0 YaaniBrowser/4.3.0.153 (Turkcell-TR) Mobile Safari/537.36

Reply:
others models (optional)

"Dalvik/1.6.0 (Linux; U; Android 4.1.2; TURKCELL T40 Build/JZO54K)"
"Dalvik/1.6.0 (Linux; U; Android 4.4.2; TURKCELL T50 Build/KVT49L)"
"Dalvik/2.1.0 (Linux; U; Android 5.0.2; TURKCELL T60 Build/LRX22G)"
"Mozilla/5.0 (Linux; Android 6.0.1; TURKCELL T70) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.74 Mobile Safari/537.36"
"Dalvik/2.1.0 (Linux; U; Android 7.1.1; TURKCELL T80 Build/NMF26F)"
"Mozilla/5.0 (Linux; Android 4.4.2; TURKCELL TURBO T50) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Mobile Safari/537.36"
"Mozilla/5.0 (Linux; Android 5.1; Turkcell_T_Tablet) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.106 Safari/537.36"
"Mozilla/5.0 (Linux; U; Android 2.1-update1; ru-ru; Turkcell T10 Build/ERE27) AppleWebKit/530.17 (KHTML, like Gecko) Version/4.0 Mobile Safari/530.17"
"Mozilla/5.0 (Linux; U; Android 2.3.6; ru-ru; Turkcell T20) Build/HuaweiU8650 AppleWebKit/533.1 (KHTML, like Gecko) Version/4.0 Mobile Safari/533.1"
"Mozilla/5.0 (Linux; U; Android 4.0.4; tr-tr; Turkcell Maxi Plus 5 Build/IMM76D) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30"
"Mozilla/5.0 (Linux; U; Android 4.0.3; tr-tr; TURKCELL MaxiPRO5 Build/HuaweiU8860) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30"

Reply:
All of them added!